shell/platform/darwin/common/framework/Headers/FlutterBinaryMessenger.h

```diff
@@ -29,6 +29,8 @@ typedef void (^FlutterBinaryReply)(NSData* _Nullable reply);
  */
 typedef void (^FlutterBinaryMessageHandler)(NSData* _Nullable message, FlutterBinaryReply reply);
 
+typedef int64_t FlutterBinaryMessengerConnection;
```

Comment:
same nit: `FlutterBinaryMessengerConnectionCount`?

Maybe a docstring about what this is for?